### ISSUE TITLE
To train the model on sloika

### ISSUE THREAD

**Issue by zhongpanwu** [nanoporetech/sloika#3]
Hi, I'm doing some research on basecaller training and I think Sloika is a good tool to start, even though it's obsolete. First I want to get it to run under my current system, Ubuntu 18.04, even though I have all the dependencies installed which include:
h5py (2.6.0)
numpy (1.23.2)
theano (0.8.2)
biopython (1.79)
fast5_reaserch (1.2.22)
pysam (0.19.1)
matplotlib (3.5.3)
But I still can't run training_network.py. The main error reported is:

Model file pretrained.pkl was neither a network nor compiled network.

I don't know what's causing this, but the error persists even though I modified helpers.py to force the network to be copied to output_file. The first thing I would like to understand is what exactly is pretrained.pkl (I can only see it as a Layers.Serial based object) and how do I reproduce it? Also, is there any other suggestion that would allow me to run Sloika.

Thanks in advance
Zhongpan

**Comment by cjw85** [comment]
You are correct, sloika is obsolete, deprecated and no longer supported. If you wish to train basecaller models I would direct you to [bonito](https://github.com/nanoporetech/bonito).

**Comment by zhongpanwu** [comment]
> You are correct, sloika is obsolete, deprecated and no longer supported. If you wish to train basecaller models I would direct you to [bonito](https://github.com/nanoporetech/bonito).

Thank you.
 I'm also interested in Bonito, but for now I'd like to start with the simplest neural networks, like scrappie. Unfortunately, all of scrappie's models are hardcoded, but I found clues to prove that sloika is used to generate scrappie models.

**Comment by cjw85** [comment]
Yes sloika and scrappie are contemporaneous. Reading the source code for scrappie should provide you with the information required to implement the same network architecture within bonito, or independently.

I'm sorry, I cannot provide any more help.
